common/src/lib.rs

```diff
@@ -1,3 +1,7 @@
+#[cfg(test)]
+#[cfg(macro_use)]
+extern crate lazy_static;
+
```

Comment:
This might not be needed since the test is including (`use`ing) it too. It's also a bit unexpected